fontique/src/font.rs

```diff
@@ -253,18 +309,21 @@ pub struct Synthesis {
 }
 
 impl Synthesis {
-    /// Returns true if any synthesis suggestions are available.
+    /// Returns `true` if any synthesis suggestions are available.
     pub fn any(&self) -> bool {
         self.len != 0 || self.embolden || self.skew != 0
     }
 
     /// Returns the variation settings that should be applied to match the
     /// requested attributes.
+    ///
+    /// When using `parley`, these can be used to create `FontVariation`
```

Comment:
Maybe? we should link to docs.rs here?

Reply:
How do other crosslinks work on docs.rs? I don't particularly like the idea of absolute links to docs.rs inside the code.

Reply:
This is https://github.com/rust-lang/rust/issues/74481
And indeed is similar to https://linebender.org/blog/doc-include/

Reply:
I left it as-is for someone else to revisit if they like.